web/gui/dashboard/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Ps from 'perfect-scrollbar';
+
+import './types/global'
+import './dashboard'
+import logo from './logo.svg'
+import './App.css'
+
+window.Ps = Ps
+
+const App: React.FC = () => { // eslint-disable-line arrow-body-style
```

Comment:
Are React.FC / React.SFC types still relevant?

Reply:
@Azein are they not? `FC` is just `FunctionComponent`, but you can use it without specifying Props, so they default to `{}`

Reply:
btw. this is create-react-app boilerplate